specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "description": "Get a list of all private offers in the given private store",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OfferListResponse"
+            }
+          },
+          "404": {
+            "description": "No private store enabled"
+          }
+        },
+        "x-ms-examples": {
```

Comment:
> "x [](start = 7, length = 3)

add paging support  (service may always return only one page.. but adding in swagger upfront is safe. Adding in future requires revving the api version, as it would be a breaking change)

Reply:
"x-ms-pageable": {
          "nextLinkName": "nextLink"
        },

---
In reply to: [368370371](https://github.com/Azure/azure-rest-api-specs/pull/8191#discussion_r368370371) [](ancestors = 368370371)

Reply:
adding nextLink for all the collection calls

---
In reply to: [368370371](https://github.com/Azure/azure-rest-api-specs/pull/8191#discussion_r368370371) [](ancestors = 368370371)

Reply:
done